UNRELEASED.md

```diff
@@ -32,3 +32,5 @@
 - Migrated tests using document.activeElement to use react-testing ([#3070](https://github.com/Shopify/polaris-react/pull/3070))
 
 ### Deprecations
+
+- Removed `centeredLayout` prop in `EmptyState`. All layouts within the new design language context will be center aligned ([#3111](https://github.com/Shopify/polaris-react/pull/3111))
```

Comment:
v5 release notes go in UNRELEASED-v5.md, this should go under the "breaking changes" heading